# Request
string hostname
string camera_name
bool pick
bool walk
bool place
int32 x_min
int32 y_min
int32 x_max
int32 y_max
---
# Result
bool success
string message
---
# Feedback
string current_state
